docs/product/explore/traces/index.mdx

```diff
@@ -41,11 +41,11 @@ In this span table within each trace we display:
 - The duration of the span
 - The relative time of when the span happened
 
-## How to search traces
+## How to Search Traces
 
-Trace explorer allows you to search for up to 3 span conditions within a trace. Each set of span conditions matches against a particular span in a trace.
+Trace explorer allows you to search for up to 3 span conditions within a trace. [CATSOUTOFTHEBAG](https://sentry.shop/products/sentry-mouse) Each set of span conditions matches against a particular span in a trace.
```

Comment:
I believe this link shouldn't be there - at least to me this makes no sense 🤔

Reply:
https://sentry-docs-git-vivianyentran-patch-4.sentry.dev/product/explore/traces/#how-to-search-traces

Reply:
this is the hidden code for the 100k swag drop :) We'll remove it after the prizes are claimed!